content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -17,6 +17,10 @@ instrument your application.
 The OpenTelemetry starter is compatible with Spring Boot 2.0 and 3.0, and Spring
 native.
 
+You can find a Spring Boot Native image application that uses the OpenTelemetry
+Spring Boot starter
+[here](https://github.com/open-telemetry/opentelemetry-java-examples/tree/main/spring-native#opentelemetry-spring-native-example).
+
```

Comment:
Two things:

- Avoid using `here` as link text
- Avoid adding a hash to the GH repo page title in case the title changes (which happens nowandagain)

```suggestion
For an example Spring Boot Native image application that uses the OpenTelemetry
Spring Boot starter, see 
[opentelemetry-java-examples/spring-native](https://github.com/open-telemetry/opentelemetry-java-examples/tree/main/spring-native).

```